service/src/intTest/resources/application.yml

```
@@ -17,19 +17,19 @@ spring:
     gateway:
       routes:
         - id: get-structured-record
-          uri: ${GPC_CONSUMER_GPC_GET_URL:http://localhost:8210}
+          uri: http://localhost:8210
           predicates:
-          - Path=${GPC_CONSUMER_GPC_STRUCTURED_PATH:/GP0001/STU3/1/gpconnect/fhir/Patient/$gpc.getstructuredrecord}
+          - Path=/{odsCode}/STU3/1/gpconnect/fhir/Patient/$gpc.getstructuredrecord
           - Method=POST
         - id: get-document
-          uri: ${GPC_CONSUMER_GPC_GET_URL:http://localhost:8210}
+          uri: http://localhost:8210
           predicates:
-          - Path=${GPC_CONSUMER_GPC_GET_DOCUMENT_PATH:/GP0001/STU3/1/gpconnect/fhir/Binary/{documentId}}
+          - Path=/{odsCode}/STU3/1/gpconnect/fhir/Binary/{documentId}
```

The documents endpoints need to be on different base paths because the underlying FHIR servers are different.

/Patient/1 on access structured is not the same patient as /Patient/1 on access documents. 

```suggestion
          - Path=/{odsCode}/STU3/1/gpconnect/documents/Binary/{documentId}
```